lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packages": [
+    "common/predictive-text",
+    "common/lexical-model-types",
+    "web"
+  ],
```

Comment:
Note:  this array is managed _**manually**_.

`lerna` will not automatically add packages to this list.  Anything _**we**_ add here, `lerna` will manage, but it will leave other packages alone.

Reply:
Would `web/README.md` be the place to document how we manually manage packages with lerna?
Along with updating new build options.

For the 🚌 factor

Reply:
I'll add something to the documentation for `verify_npm_setup` in its source script for this; that'd be a far better place.